adal/build.gradle

```diff
@@ -42,13 +42,14 @@ android {
     }
 
     lintOptions {
-        abortOnError false
+        abortOnError true
         disable 'MissingPermission'
         disable 'LongLogTag'
         disable 'DefaultLocale'
         disable 'UnusedResources'
         disable 'GradleDependency'
         disable 'GradleOverrides'
+        disable 'OldTargetApi'
```

Comment:
@heidijinxujia Anything you put in this file is supressed across the code base
@weijjia We should know when we are suppressing this, putting it here will hide it forever. So prefer doing it whereever required.